docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/StochasticDualCoordinateAscent.cs

```diff
@@ -1,8 +1,12 @@
-﻿using Microsoft.ML;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
 
 namespace Samples.Dynamic.Trainers.Regression
 {
-    public static class StochasticDualCoordinateAscent
+    public static class Sdca
```

Comment:
we kept going back and forth for sdca. The final version is using the acronym. please also change the filenames to Sdca (don't forget to update xmls referring this file)

Reply:
Sure.

---
In reply to: [275564867](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275564867) [](ancestors = 275564867)